### ISSUE TITLE
Add (13C)carbon dioxide and (13C)hydrogencarbonate

### ISSUE THREAD

**Issue by matthiaskoenig** [ebi-chebi/ChEBI#3408]
Many C13 tracer experiments are using 13CO2 for the detection. Examples given are liver function tests based on [13C]methacetin, [13C]caffeine or [13C]galactose.

Could you please add

`(13C)carbon dioxide` as a child of
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:16526
"A 13C-modified compound that is carbon dioxide in which the carbon is present as its 13C isotope."

and 
`(13C)hydrogencarbonate` as a child of
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:17544
"A 13C-modified compound that is hydrogencarbonate in which the carbon is present as its 13C isotope."

Analogue to how these tracer substances are handled for instance for 
`(13C)urea`
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:32285

This should basically be a child of the original substance.
Best Matthias

**Comment by G-Owen** [comment]
Hi Matthias,

We can certainly add these compounds. However, I am not entirely convinced that we can classify (say) 13CO2 (CHEBI:139538) is_a CO2 (CHEBI:16526), since our entry for CO2 (CHEBI:16526) says that it has a molecular weight of 44.01, whereas 13CO2 actually has a molecular weight of 45.00. 

I suspect that what we really need for linking isotopically modified entries to their unmodified ('naturally occurring mix of isotopes') version is one or two pairs of new relationships in the ontology: 'is_isotopomer_of' and its reverse 'has_isotopomer'  
and/or 'is_isotopologue_of' and its reversE  'has_isotopologue'
(see https://en.wikipedia.org/wiki/Isotopologue and https://en.wikipedia.org/wiki/Isotopomers).

Let me know what you think!

Best regards,
Gareth

**Comment by matthiaskoenig** [comment]
Yes, sounds good to me. I just need a way to annotate these substances in
computational models. The exact relationship to the original unmodified
compound is not so important as long as there is a link and the isotopomers
are handled in a consistant manner.

**Comment by matthiaskoenig** [comment]
I am very happy with the proposed solution using the relationships:
`is_isotopomer_of` and its reverse `has_isotopomer` and/or `is_isotopologue_of` and its reverse `has_isotopologue`
It would be great if these substances could be added. There will definitely be more of these in upcoming models of mine like [13C]-ocanoate, [13C]-phenylalanine, [13C]-caffeine, [13C]-galactose, ... So it makes sense to add a relationship for these and handle them consistently.

Let me know if you need any input from my side.
Best Matthias
